manager/src/main/java/org/openremote/manager/dashboard/DashboardStorageService.java

```diff
@@ -292,17 +285,11 @@ public Dashboard update(Dashboard dashboard, String realm, String userId) throws
         if(userId == null) {
             throw new IllegalArgumentException("No userId is specified.");
         }
-        // Query the dashboards with the same ID (which is only 1), and that userId is able to EDIT
+        // Query the dashboards with the same ID (which is only 1)
```

Comment:
I've only walked the code path here in github review but it looks like any user can update/delete another user's dashboard if they have the ID

Reply:
Entity version override shouldn't be done without a good reason, it is the protection against merge conflicts (i.e. if someone updated before you)

Reply:
Below the constructor of the `DashboardQuery`, it's calling `this.query(query, userId)`.
This function queries dashboards based on `userId`, automatically checking access for this user.
It's used everywhere, also in `GET /dashboard` for example.

Reply:
> Entity version override shouldn't be done without a good reason, it is the protection against merge conflicts (i.e. if someone updated before you)

https://github.com/openremote/openremote/issues/1836 now tracks this issue.